conda_build/conda_interface.py

```diff
@@ -123,3 +125,40 @@ def which_prefix(path):
             # we cannot chop off any more directories, so we didn't find it
             return None
         prefix = dirname(prefix)
+
+
+class NodeType(Enum):
```

Comment:
Above there's a line

    if parse_version(conda.__version__) >= parse_version("4.2"):

We should also have a section for `4.3` where we actually do the real imports from `conda/exports.py`.  Then add this new stuff under the `else` after `4.3`.  That way it's easy to rip out in the future.